drop-in/src/main/java/com/adyen/checkout/dropin/ui/paymentmethods/PaymentMethodAdapter.kt

```diff
@@ -117,6 +117,7 @@ internal class PaymentMethodAdapter @JvmOverloads constructor(
             }
         }
 
+        @SuppressLint("StringFormatInvalid")
```

Comment:
Why are these suddenly needed?

Reply:
I found an error over them that for example instead of using `context.getString(R.string.card_number_4digit, model.lastFour)` for example it recommends to change it to `String.format`

Reply:
after some research for this lint highlight is looks like a lint bug and when I just cleaned project it went away. Will remove it.